src/app/login/components/Login.tsx

```diff
@@ -63,20 +98,62 @@ export const LoginPage = () => {
                           placeholder="0300-0000000"
                           {...field}
                           value={field.value}
-                          onChange={(e) =>
-                            field.onChange(formatPhoneNumber(e.target.value))
-                          }
+                          data-testId="phoneNumber"
+                          onChange={(e) => field.onChange(e.target.value)}
```

Comment:
i think we should not need because react-form-hook handle it if we want to custom need of phonenumber then we can use it